<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>697</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Menu">
       <widget class="QLabel" name="label_MenuParameters">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>40</y>
          <width>681</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Меню выбора параметров рисунка</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Menu_Draw">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>550</y>
          <width>311</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Нарисовать</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_Menu_Color">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>130</y>
          <width>231</width>
          <height>71</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Желтый</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Белый</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Красный и тд</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_Menu_Color">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>140</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Цвет дома:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Menu_Time">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>250</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Время суток:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_Menu_Time">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>250</y>
          <width>231</width>
          <height>71</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Ночь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>День</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_Menu_Stars">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>360</y>
          <width>211</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Кол-во звезд:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Menu_Error">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>550</y>
          <width>591</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_Stars">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>370</y>
          <width>231</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Image">
       <widget class="paint_widget" name="widget_Image" native="true">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>29</y>
          <width>1131</width>
          <height>501</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_Image_PixMap">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>200</width>
           <height>200</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="images.qrc">:/Images/Moon.jpg.png</pixmap>
         </property>
        </widget>
        <widget class="QLabel" name="label_Image_Cloud1">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>20</y>
           <width>200</width>
           <height>150</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="images.qrc">:/Images/Cloud1.png.png</pixmap>
         </property>
        </widget>
        <widget class="QLabel" name="label_Image_Cloud2">
         <property name="geometry">
          <rect>
           <x>840</x>
           <y>30</y>
           <width>200</width>
           <height>200</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="images.qrc">:/Images/Cloud2.png</pixmap>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_Image_BackToMenu">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>550</y>
          <width>291</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>В меню</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1190</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>paint_widget</class>
   <extends>QWidget</extends>
   <header location="global">paint_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
